Formula/b2sum.rb

```diff
@@ -0,0 +1,20 @@
+class B2sum < Formula
+  desc "BLAKE2 b2sum reference binary"
+  homepage "https://github.com/BLAKE2/BLAKE2"
+  url "https://github.com/BLAKE2/BLAKE2/archive/20160619.tar.gz"
+  sha256 "cbac833ccae56b5c6173dbeaf871aa99b32745cf7a994c7451d4533ecda55633"
+
+  def install
+    cd "b2sum" do
+      # https://github.com/BLAKE2/BLAKE2/issues/25
+      system "make", "NO_OPENMP=1"
+      bin.install "b2sum"
+      man1.install "b2sum.1"
+    end
+  end
+
+  # brew test b2sum
+  test do
+    assert_equal(shell_output("echo 'abc' | b2sum -").strip, "426526ad3aa99ac6bbb98555c6c30b5a55189d0d7537d9214053b886f365a84bd7e68c91b544a8f485fa842c0ea9fbc4fb92277290264e685eed9e67b7621fc9  -")
```

Comment:
The expected value should be the first argument to `assert_equal`, you should use `pipe_output` instead of `shell_output`, and you need to reference the `b2sum` binary with its full path. In summary, something like this will probably work:

```ruby
assert_equal "426526ad3aa99ac6bbb98555c6c30b5a55189d0d7537d9214053b886f365a84bd7e68c91b544a8f485fa842c0ea9fbc4fb92277290264e685eed9e67b7621fc9  -",
             pipe_output("#{bin}/b2sum -", "abc").chomp
```